rental.rb

```diff
@@ -0,0 +1,10 @@
+class Rental
+  attr_accessor :date
+  attr_reader :person, :book
+
+  def initialize(date, person, book)
+    @date = date
+    @person = person
+    @book = book
+  end
+end
```

Comment:
kindly implement the belongs_to relationship for rentals and book, rentals and person👍.
<img width="553" alt="image" src="https://github.com/mumo-esther/Library/assets/64643995/42038b2e-c116-4bf4-a6f6-dca7b20961c4">


we can achieve it this way👍.

```suggestion
  def initialize(date, person, book)
    @date = date
    @person = person
    person.rentals << self
    @book = book
    book.rentals << self
  end
end
```